tests/actions/ReportTest.js

```diff
@@ -275,7 +275,7 @@ describe('actions/Report', () => {
             .then(() => {
                 // The report will be read
                 expect(ReportUtils.isUnread(report)).toBe(false);
-                expect(moment.utc(report.lastReadTime).valueOf()).toBeGreaterThanOrEqual(moment.utc(currentTime).valueOf());
+                expect(utcToZonedTime(report.lastReadTime, 'UTC').getTime()).toBeGreaterThanOrEqual(utcToZonedTime(currentTime, 'UTC').getTime());
```

Comment:
should we add 'UTC' to const? 🤔

Reply:
I'd say unless it's a strictly typed string but +1!